playwright/README.md

```diff
@@ -227,6 +227,107 @@ new AxeBuilder(page)
         .setLegacyMode(true);
 ```
 
+## Limit Frame Testing
```

Comment:
We should also include the new include with just a single string in the sections above where we talk about a list of strings of just one (saying they are equivalent).